Makefile

```
@@ -12,7 +12,9 @@ help:
 	@echo "make lint              Run the code linter(s) and print any warnings"
 	@echo "make format            Correctly format the code"
 	@echo "make checkformatting   Crash if the code isn't correctly formatted"
-	@echo "make test              Run the unit tests"
+	@echo "make test              Run the all tests"
```

```suggestion
	@echo "make test              Run the unit tests"
```

Saying "run all tests" is misleading because this command doesn't run _all_ of the Python tests. Only the Python _unit_ tests. There are some other integration/E2E tests as well.

---

Good observation. It was fixed.